tests/unit/utils/test_rsax931.py

```diff
@@ -174,49 +170,86 @@ def test_find_libcrypto_aix(self):
                 fnmatch.fnmatch(lib_path, "/opt/freeware/lib/libcrypto.so*")
             )
 
-    @skipIf(not salt.utils.platform.is_darwin(), "Host OS is not Darwin-like or macOS.")
+    @patch.object(salt.utils.platform, "is_darwin", lambda: True)
     @patch.object(platform, "mac_ver", lambda: ("10.14.2", (), ""))
     @patch.object(glob, "glob", lambda _: [])
-    def test_find_libcrypto_with_system_and_not_catalina(self):
+    def test_find_libcrypto_with_system_before_catalina(self):
         """
         Test _find_libcrypto on a Catalina-like macOS host, simulate
```

Comment:
Can you update the comment to reflect this tests before catalina and not on a catalina macos host.